_posts/kubernetes/1200-09-01-K8-general-kubernetes-questions.md

```diff
@@ -35,9 +35,41 @@ It is possible to deploy an ingress resource in the same way you can deploy any
 
+## How do we handle deployment in multiples cluster? For example, if our environment includes more than one k8s cluster
+With the account concept in K8s, an account is just a k8s cluster, when you configure an account you configure it to use an specific kube config file, this define how we communicate and authenticate on our cluster, if you have multiple cluster you can configure multiple accounts within spinnaker using different kube config files that have the address for the API, when you set the deployment step you setup a different account to deploy to and spinnaker will use the credentials found in the kube config.
 
+## There are two different types of pipelines : Startegy and Pipeline , what's the difference.
+A pipeline is generally just the way your application rolls out, that would be like saying deploying to a particular environment manual judgment deploying into another, we do things like kicking of jenkins jobs. Strategy is kind of a way of taking all of the different pieces of that pipeline and being able to select them as a way you want to deploy. 
 
+## It seems that Spinnaker is polling data from Kubernetes. How often is this data refreshed?
+By default spinnaker tries to pull every 30 seconds, but you can configure this time.
 
+## How's the performance of Spinnaker when it's keeping track of hundreds of pipelines?
+Spinnaker was build to run at scale, hundreds of pipelines are safe for spinnaker but comes with the caviat that you have at some extend manage and scale spinnaker controlers. We have at spinnaker.io a little guide at what compponents to scale horizontall, depending on how your work looks like, in the case of pipelines there is a component called front50, you have to probably add more replicas of that so you don´t run into issues when running large number of pipelines
 
+## is the V2 provider still in beta and subject to significant changes?
+Since release 1.10 it's out of Beta, there are still many large features that will get added to the provider. For example Traffic Management for Red/Black, Rolling Red/Black, Canary, etc. Also added label manipulation and Istio.
+Changes to the Managed Pipeline Templates to allow operators to define custom pipelines that the users can use insted of the users making pipelines by hand over and over again.
+Artifact provenance for software supply chain management, this will allow you to ask questions like, which commitment made into production since the last release, what changes in my config map trigger this release.
 
+## Any plans to move from kubectl to K8S APIs?
+There is no plans at the moment, one of the reasons we use kubectl specially for deployment is that there is some merging loginc that is build into kubectl that we are taking advantage of, so the primary way that the people interact with the k8s API today is through kubectl.
+As it stand using any of the apis beside the oficial one and kubectl is very dificult and odds they are largely unsupported
```

Comment:
s/stand/stands
s/apis/APIs/

period after unsupported.

Reply:
check. Also all other missing periods at the end.